woocommerce/woocommerce
# Only trim strings in multiselect display options

Fixes #19120 

To test:
- Get and activate [Paystack Gateway](https://wordpress.org/plugins/woo-paystack/).
- Add an aditional gateway in the Paystack settings
- In the aditional gateway verify the Payment Icons setting works properly.
- In the shipping zones settings add a new region and verify the indentation looks good and buttons work properly (to make sure no behavior was regressed)
